examples/external-data/src/mock-external-data-service/service.ts

```diff
@@ -110,25 +159,25 @@ export async function initializeExternalDataService(props: ServiceProps): Promis
 	 * }
 	 * ```
 	 */
-	expressApp.post("/register-for-webhook", (request, result) => {
-		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-		const subscriberUrl = request.body?.url as string;
-		if (subscriberUrl === undefined) {
-			const errorMessage = 'No subscription URL provided. Expected under "url" property.';
-			console.log(formatLogMessage(errorMessage));
-			result.status(400).json({ message: errorMessage });
-		} else if (isWebUri(subscriberUrl) === undefined) {
-			const errorMessage = "Provided subscription URL is invalid.";
-			console.log(formatLogMessage(errorMessage));
-			result.status(400).json({ message: errorMessage });
-		} else {
-			// TODO: use a query string parser here instead of this hacky lookup of '=externalTaskListId'
-			// Tried using node:querystring and node:url but that is not allowed by
-			// the rules and haven't found another one that works in a simple search so far.
-			// This method is incredibly brittle and will break if we add any other qs param
-			const externalTaskListId = subscriberUrl.slice(
-				subscriberUrl.indexOf("externalTaskListId=") + "externalTaskListId=".length,
-			);
+	expressApp.post("/register-for-webhook", (request: RegisterWebhookRequest, result) => {
+		try {
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
This shouldn't be necessary anymore, I think?

Reply:
Yep, done in commit #a8d6dd5